.env.example

```
@@ -6,76 +17,211 @@ NEXT_PUBLIC_MARKETING_URL=http://localhost:3000
+#   REDIS_REST_TOKEN
+#     -> UPSTASH_REDIS_REST_TOKEN / UPSTASH_REDIS_REST_LOCKER_TOKEN
+#   POSTGRES_USER / POSTGRES_PASSWORD / POSTGRES_DB
+#     -> the credentials in both POSTGRES_PRISMA_URL* values
+#
+# Changing these after the first `docker compose up` does NOT update the
+# existing volumes — Postgres and MinIO only read them when initialising
+# empty data directories. Set them before the first start, or recreate the
+# volumes (`docker compose down -v`, which destroys all data).
+# =============================================================================
+POSTGRES_USER=papermark
+POSTGRES_PASSWORD=papermark
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)